chapter02/worksheets/solution2.1.7.2_AR.scala

```diff
@@ -19,6 +19,6 @@ case class Solution2_1_7_2_AR_check(i: Int, k: Int, j: Int) {
 // Check the value of expected
 val check_solution2_1_7_2_AR = expected.map({case (i, j, k) => ((i +  4 * k + 9 * j), (i * j * k)) }).filter{ case (x, y) => x <  y }
 
-printf("check_solution2_1_7_2_AR: this should be empty: %s", check_solution2_1_7_2_AR)
+printf("\ncheck_solution2_1_7_2_AR: this should be empty: %s", check_solution2_1_7_2_AR)
```

Comment:
This is a minor comment but I would suggest to add `\n` always **at the end** of the string, not at the beginning. That is the usual convention. For example, `println` automatically adds `\n` after the string. If you add `\n` at the beginning then sometimes the output will still run into something printed after it, and sometimes you will have extra empty lines.

Reply:
Done. Thanks for all your care.